src/Common/src/System/Net/_LazyAsyncResult.cs

```diff
@@ -430,15 +430,18 @@ protected virtual void Complete(IntPtr userToken)
         }
 
         // Only called in the above method
-        private void WorkerThreadComplete(object state)
+        private static void WorkerThreadComplete(object state)
```

Comment:
This change is fine, but if you wanted to leave this method as it was, you could instead have just made the call site above use:
```C#
s => ((LazyAsyncResult)s).WorkerThreadComplete(), this
```